lib/graphql/tracing/data_dog_tracing.rb

```diff
@@ -15,28 +15,32 @@ class DataDogTracing < PlatformTracing
       }
 
       def platform_trace(platform_key, key, data)
-        service = options.fetch(:service, 'ruby-graphql')
+        tracer.trace(platform_key, service: service_name) do |span|
+          span.span_type = 'custom'
 
-        pin = Datadog::Pin.get_from(self)
-        unless pin
-          pin = Datadog::Pin.new(service)
-          pin.onto(self)
-        end
-
-        pin.tracer.trace(platform_key, service: pin.service) do |span|
           if key == 'execute_multiplex'
-            span.resource = data[:multiplex].queries.map(&:selected_operation_name).join(', ')
+            operations = data[:multiplex].queries.map(&:selected_operation_name).join(', ')
+            span.resource = operations if operations.length.empty?
```

Comment:
What is `operations.length.empty?`? Should it be `.zero?`?

```ruby 
# $ irb
irb(main):001:0> 0.empty?
# NoMethodError: undefined method `empty?' for 0:Fixnum
# 	from (irb):1
# 	from /usr/bin/irb:12:in `<main>'
irb(main):002:0> 0.zero?
# => true
```

Reply:
Oh man... I must have fat fingered that one. I meant `operations.empty?` because `operations` should be a String. I'll update this.